src/main/java/book/Book.java

```diff
@@ -0,0 +1,62 @@
+package book;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import book.command.Command;
+import book.exception.BookException;
+import book.gui.Gui;
+import javafx.application.Application;
+import javafx.application.Platform;
+
+/**
+ * Main class of {@code Book}.
+ */
+public class Book {
```

Comment:
Really excellent effort! It's good you've managed to document all your code clearly and managed to follow the coding standards consistently through all your classes!